src/benchq/compilation/graph_sim_mini.jl

```diff
@@ -278,24 +290,35 @@ function get_icm(circuit, n_qubits::Int, with_measurements::Bool=false)
     ops = get_op_list()
     curr_qubits = n_qubits
     for op in circuit
-        op_index = get_op_index(ops, op)
-        if single_qubit_op(op_index)
-            push!(compiled_circuit, ICMOp(code_list[op_index], qubit_map[get_qubit_1(op)+1]))
-        elseif double_qubit_op(op_index)
-            push!(compiled_circuit,
-                  ICMOp(code_list[op_index],
-                        qubit_map[get_qubit_1(op)+1], qubit_map[get_qubit_2(op)+1]))
-        elseif decompose_op(op_index)
-            # Note: these are currently all single qubit gates
+        if occursin("ResetOperation", pyconvert(String, op.__str__()))
             original_qubit = get_qubit_1(op)
             compiled_qubit = qubit_map[original_qubit+1]
             qubit_map[original_qubit+1] = new_qubit = curr_qubits
             curr_qubits += 1
 
-            push!(compiled_circuit, ICMOp(CNOT_code, compiled_qubit, new_qubit))
             with_measurements &&
                 push!(compiled_circuit,
                       ICMOp(code_list[op_index]+MEASURE_OFFSET, compiled_qubit, new_qubit))
```

Comment:
Delete this lines since we don't need the measurement result.

Reply:
done!